### PR TITLE
global: format code with yapf

### PR DESCRIPTION
**Pull request by spirosdelviniotis**
* Format python code in a common pattern provided by yapf configuration.

Signed-off-by: Spiros Delviniotis <spyridon.delviniotis@cern.ch>

### CODE REVIEW

File: inspirehep/dojson/hep/fields/bd01x09x.py [hunk 1]
```diff
@@ -90,7 +88,7 @@
 @hep.over('isbns', '^020..')
 @utils.for_each_value
 def isbns(self, key, value):
-    "ISBN, its medium and an additional comment."""
+    "ISBN, its medium and an additional comment." ""
```
Comment (jmartinm) [inline inspirehep/dojson/hep/fields/bd01x09x.py]: This looks like a mistake

File: inspirehep/dojson/conferences/fields/bd1xx.py [hunk 2]
```diff
@@ -155,7 +154,4 @@ def _last_is_incomplete(series, key):
 @utils.for_each_value
 def short_description(self, key, value):
     """Conference short_description."""
-    return {
-        'value': value.get('a'),
-        'source': value.get('9')
-    }
+    return {'value': value.get('a'), 'source': value.get('9')}
```
Comment (jmartinm) [inline inspirehep/dojson/conferences/fields/bd1xx.py]: I actually like the multiline dictionary.

**Comment by jacquerie** [comment]
Putting this back to WIP because we will need it after the current sprint. No action needed from you, @spirosdelviniotis.

**Comment by kaplun** [comment]
BTW Invenio core is also looking into yapf: https://github.com/inveniosoftware/requirements-builder/issues/19

**Comment by jacquerie** [comment]
> BTW Invenio core is also looking into yapf: inveniosoftware/requirements-builder#19

Nice. I also like their defaults.

**Comment by spirosdelviniotis** [comment]
Here is the result of invenio defaults for yapf.

File: inspirehep/bat/pages/holding_panel_author_detail.py [hunk 3]
```diff
@@ -81,36 +113,53 @@ def _load_submitted_record():
...
         return 'Accepted as Non-CORE' in WebDriverWait(Arsenic(), 10).until(
-            EC.visibility_of_element_located((By.XPATH, '//div[@class="alert ng-scope alert-accept"]'))).text
+            EC.visibility_of_element_located((
+                By.XPATH, '//div[@class="alert ng-scope alert-accept"]'
+            ))
+        ).text
 
-    Arsenic().find_element_by_xpath('//button[@class="btn btn-success"]').click()
+    Arsenic().find_element_by_xpath('//button[@class="btn btn-success"]'
+                                    ).click()
```
Comment (david-caro) [inline inspirehep/bat/pages/holding_panel_author_detail.py]: this looks weird

File: inspirehep/bat/pages/holding_panel_author_list.py [hunk 4]
```diff
@@ -37,27 +37,39 @@
...
         return (
-            'CERN' in record and
-            'ACC-PHYS' in record and
-            'ASTRO-PH' in record and
-            'Twain, Mark' in record and
-            'inspire:uid:1' in record and
-            'admin@inspirehep.net' in record
+            'CERN' in record and 'ACC-PHYS' in record and
+            'ASTRO-PH' in record and 'Twain, Mark' in record and
+            'inspire:uid:1' in record and 'admin@inspirehep.net' in record
```
Comment (david-caro) [inline inspirehep/bat/pages/holding_panel_author_list.py]: this is easy to misread

File: inspirehep/dojson/hep/fields/bd90x99x.py [hunk 5]
```diff
@@ -54,25 +52,18 @@ def get_value(value):
...
+                       rb.add_raw_reference, source='dojson'
+                   )), ('1', rb.set_texkey), ('u', rb.add_url),
+                   ('r', rb.add_report_number), ('s', rb.set_pubnote),
+                   ('p', rb.set_publisher), ('y', rb.set_year),
+                   ('i', rb.add_uid), ('b', rb.add_uid), ('a', rb.add_uid),
+                   ('c', rb.add_collaboration), ('q', rb.add_title),
+                   ('t', rb.add_title), ('h', rb.add_refextract_authors_str),
+                   ('e', partial(
+                       rb.add_author, role='ed.'
+                   ))]
```
Comment (david-caro) [inline inspirehep/dojson/hep/fields/bd90x99x.py]: This also is harder to read now

File: inspirehep/modules/authors/rest/citations.py [hunk 6]
```diff
@@ -50,13 +50,11 @@ def serialize(self, pid, record, links_factory=None):
...
-                "authors.recid",
-                "control_number",
-                "self",
-            ]
-        )
+        }).params(_source=[
+            "authors.recid",
+            "control_number",
+            "self",
+        ])
```
Comment (david-caro) [inline inspirehep/modules/authors/rest/citations.py]: Feels strange that one indentation level does not match one logical level to me, for me this is clearer:

```python
).params(
    _source=[
        "authors.recid",
        "control_number",
        "self",
    ]
)
```

Even if it uses more vertical space.

File: inspirehep/modules/authors/rest/publications.py [hunk 7]
```diff
@@ -96,14 +97,17 @@ def serialize(self, pid, record, links_factory=None):
             try:
                 publication['journal'] = {}
                 publication['journal']['title'] = result_source.get(
-                    'publication_info', [])[0]['journal_title']
+                    'publication_info', []
```
Comment (david-caro) [inline inspirehep/modules/authors/rest/publications.py]: Hmmm, if splitting a function on multiple lines, I think it's a lot more readable if you put each param on it's own line.

Comment (david-caro) [inline inspirehep/modules/authors/rest/publications.py]: For example:

```python
something(
    'string, with commas, and', something, 'that', (' makes, (it' + really, really), hard, ')to read',
) 
# vs
something(
    'string, with commas, and',
    something,
    'that',
    (' makes, (it' + really, really),
    hard,
    ')to read',
)
```

On the first you have completely no idea how many params are there even with syntax highlight.

File: inspirehep/modules/authors/tasks.py [hunk 8]
```diff
@@ -164,14 +165,14 @@ def curation_ticket_context(user, obj):
...
     )
     return dict(
         email=user.email,
         object=obj,
         recid=recid,
         subject=subject,
         record_url=record_url,
-        user_comment=obj.extra_data.get('formdata', {}).get('extra_comments', ''),
+        user_comment=obj.extra_data.get('formdata', {}
+                                        ).get('extra_comments', ''),
```
Comment (david-caro) [inline inspirehep/modules/authors/tasks.py]: strange that sometimes it decides to indent with 4 spaces, and sometimes it just goes all the way to the opening char. I prefer 4 spaces so it always matches the logical nested level.

File: inspirehep/modules/authors/views/publications.py [hunk 9]
```diff
@@ -51,17 +49,14 @@ def get_publications():
...
         _source=[
-            'accelerator_experiments',
-            'control_number',
-            'earliest_date',
-            'facet_inspire_doc_type',
-            'publication_info',
-            'titles',
-            'keywords'
+            'accelerator_experiments', 'control_number', 'earliest_date',
+            'facet_inspire_doc_type', 'publication_info', 'titles', 'keywords'
```
Comment (david-caro) [inline inspirehep/modules/authors/views/publications.py]: and here uses the extra newline.... strange

**Comment by spirosdelviniotis** [comment]
Now setted column limit to 99 to see possible changes.

**Comment by jacquerie** [comment]
Now that https://github.com/inspirehep/inspire-next/pull/2230 was merged, the idea is to use this to solve `E121`, `E123`, `E126`, `E128`, and `E501` automatically using `yapf`.

**Comment by jacquerie** [comment]
As evidenced by the huge amount of conflicts, this needs to happen when the waters are still, otherwise it's going to invalidate all the other PRs. Obviously `yapf` can then be run on those PR, but it's extra work I would like to minimize. Let's do this when we reach 10 open PRs.

**Comment by jacquerie** [comment]
Closed in favor of https://github.com/inspirehep/inspire-next/pull/2788. There's no longer that many style violations to require using an automated tool to fix them.

**Pull request closed without merging**